read-fonts/src/lib.rs

```diff
@@ -357,9 +387,23 @@ impl<'a> FontRef<'a> {
             })
     }
 
+    /// Returns an iterator over all of the available fonts in
+    /// the given font data.
+    pub fn all_in(
```

Comment:
maybe... `iter_all`? `iter_from_bytes`?

Reply:
totally open to suggestions on this one. `iter_all` seems fine to me

Reply:
`all_in` because using fonts is a gamble.

`iter_all` feels a touch redundant. I would expect `iter` to iterate all of them unless I filtered or limited or something.

`fonts(data)`, `iter(data)` ... even `iter_all(datat)` all seem fine.

Reply:
> `all_in` because using fonts is a gamble.

🤣 

> `iter_all` feels a touch redundant. I would expect `iter` to iterate all of them unless I filtered or limited or something.
> 
> `fonts(data)`, `iter(data)` ... even `iter_all(datat)` all seem fine.

I think I'll just go with `fonts(data)`. Seems nice and simple.